LabExT/View/Movement/MovementWizard.py

```diff
@@ -346,15 +346,15 @@ def build(self, frame: Type[CustomFrame]) -> None:
         ).pack(side=TOP, fill=X)
 
         if not self.mover.stage_classes:
-            Label(frame, text="No stage classes found!").pack(side=TOP, fill=X)
+            Label(frame, text="No stage classes found!", foreground="#FF3333").pack(side=TOP, fill=X)
 
-        for stage_cls in self.mover.stage_classes:
+        for stage_name, stage_cls in self.mover.stage_classes.items():
             stage_driver_frame = Frame(frame)
             stage_driver_frame.pack(side=TOP, fill=X, pady=2)
 
             Label(
                 stage_driver_frame,
-                text=f"[{stage_cls.__name__}] {stage_cls.meta.description}"
+                text=f"[{stage_name}] {stage_cls.meta.description}"
```

Comment:
My stages loaded from an addon path are shown here (drivers are loaded as well), but they are not listed in the stage assignment step afterwards. I used the dummy stage from the core loaded from my addon path.

Reply:
Yes, there was an error with the stage discovery. I changed it now, thanks! Can you check again if it works now?